src/datachain/remote/studio.py

```diff
@@ -158,15 +159,14 @@ def _send_request_msgpack(
             message = content.get("message", "")
         return Response(response_data, ok, message)
 
-    @retry_with_backoff(retries=5)
+    @retry_with_backoff(retries=3, errors=(HTTPError, Timeout))
     def _send_request(
         self, route: str, data: dict[str, Any], method: Optional[str] = "POST"
     ) -> Response[Any]:
         """
         Function that communicate Studio API.
         It will raise an exception, and try to retry, if 5xx status code is
-        returned, or if ConnectionError or Timeout exceptions are thrown from
-        requests lib
+        returned, or if Timeout exceptions is thrown from the requests lib
```

Comment:
[C]: removed `ConnectionError` from the one we retry on. It is usually Proxy, or something SSL - don't think we can recover from those